### PR TITLE
closes #[5] Feature/#2 login

### PR DESCRIPTION
**Pull request by gdmeridabit**
#card-20700538
#card-20700544

https://github.com/gdmeridabit/blog/projects/1#card-20700538
https://github.com/gdmeridabit/blog/projects/1#card-20700544

### CODE REVIEW

**Comment by andy1dx** [comment]
@gdmeridabit Add this comment in front of the description

closes #[issues_number]
so it will auto closes issues

File: public/css/app.css [hunk 1]
```diff
@@ -2119,11 +2119,11 @@ select.form-control[multiple], select.form-control[size], textarea.form-control
     padding: .375rem .75rem;
     font-size: .9rem;
     line-height: 1.6;
-    border-radius: .25rem;
+    border-radius: .255rem;
     transition: color .15s ease-in-out, background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out
 }
```
Comment (hugh-n) [inline public/css/app.css:2124]: Please order CSS code from A to Z as noted in the Development Rules.

border-radius: .255rem;
font-size: .9rem;
line-height: 1.6;
...

File: public/css/app.css [hunk 2]
```diff
@@ -2119,11 +2119,11 @@ select.form-control[multiple], select.form-control[size], textarea.form-control
...
 
-@media screen and (prefers-reduced-motion: reduce) {
+@media screen and (max-width: 768px) {
```
Comment (hugh-n) [inline public/css/app.css:2126]: You seem to have multiple media queries with the same condition. 

@media screen and (max-width: 768px).

You should combine the code of each into one query instead.

File: resources/views/layouts/app.blade.php [hunk 3]
```diff
@@ -8,34 +8,41 @@
...
+        <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <!--    <div class="navbar-collapse collapse w-100 order-1 order-md-0 dual-collapse2">-->
+    <!--        <ul class="navbar-nav mr-auto">-->
+    <!--            <li class="nav-item active">-->
+    <!--                <a class="nav-link" href="#">Left</a>-->
+    <!--            </li>-->
+    <!--        </ul>-->
+    <!--    </div>-->
```
Comment (hugh-n) [inline resources/views/layouts/app.blade.php]: Please delete the commented out HTML if its not needed.

**Review by hugh-n — CHANGES_REQUESTED** [review]
Please check the comments I made and fix them. I also noticed that the branch can be merged even without review. Please change the branch settings to require review first before merging.

File: public/css/app.css (hunk 2 above)
Comment (gdmeridabit) [reply public/css/app.css:2126]: this are code generated when I install the bootstrap. I just change the media queries to max-width tho.

File: public/css/app.css (hunk 1 above)
Comment (gdmeridabit) [reply public/css/app.css:2124]: this is also code generated

**Review by hugh-n — APPROVED** [review]

File: public/css/app.css (hunk 2 above)
Comment (hugh-n) [reply public/css/app.css:2126]: Got it. In that case, leave it as it is.

**Pull request merged**